### PR TITLE
 fixed: Token for free users not found

### PR DESCRIPTION
**Pull request by snoopyf**
Fix from realvito in kodinerds forum (https://www.kodinerds.net/index.php/Thread/57646-Release-TVNow-RTL-Watchbox-Vox-NTV-NOWUS-RTL2-Nitro/?postID=536927#post536927)

### CODE REVIEW

File: tvnow.py [hunk 1]
```diff
@@ -95,17 +95,18 @@ def getToken(self):
...
-        jsName = ""
-        if m:
-            jsName = m.group(1)
-        else:
+        try: jsName = re.findall(r'<script src="(main\-[A-z0-9]+\.[A-z0-9]+\.js)"', r.text, re.S)[-1]
+        except: jsName = ""
+        #m = re.search(r'<script src="(main\-[A-z0-9]+\.[A-z0-9]+\.js)"', r.text)
+        #jsName = ""
+        #if m:
+            #jsName = m.group(1)
```
Comment (TehTux) [inline tvnow.py]: Please remove the comments!

File: tvnow.py [hunk 2]
```diff
@@ -95,17 +95,18 @@ def getToken(self):
...
+        if jsName == "":
             xbmcgui.Dialog().notification('Fehler GetToken', 'JS not found', icon=xbmcgui.NOTIFICATION_ERROR)
             return "0"
```
Comment (TehTux) [inline tvnow.py]: Can be summarized to

``` 
except:
    xbmcgui.Dialog().notification('Fehler GetToken', 'JS not found', icon=xbmcgui.NOTIFICATION_ERROR)
    return "0"
```

Comment (TehTux) [inline tvnow.py:101]: L99 - 106 can be summarized to

``` 
except:
    xbmcgui.Dialog().notification('Fehler GetToken', 'JS not found', icon=xbmcgui.NOTIFICATION_ERROR)
    return "0"
``` 

and please remove the comments.

Comment (snoopyf) [reply tvnow.py:101]: OK thx for your comments, I'll change this and create a new Pull Requests :)

**Pull request closed without merging**